backends/tfhe-cuda-backend/cuda/include/integer.h

```diff
@@ -9,7 +9,12 @@
 #include <functional>
 
 enum OUTPUT_CARRY { NONE = 0, GENERATED = 1, PROPAGATED = 2 };
-enum SHIFT_TYPE { LEFT_SHIFT = 0, RIGHT_SHIFT = 1 };
+enum SHIFT_TYPE {
```

Comment:
Maybe the enum could be renamed `SHIFT_ROTATE_TYPE`

Reply:
Wdyt about `SHIFT_OR_ROTATE_TYPE`? The idea is that this type should define if the op is a shift or rotation, so I miss an "OR" in there.

Reply:
Sounds perfect